introduction/getting-started/index.md

```diff
@@ -0,0 +1,96 @@
+---
+layout:   default
+class:    markdown
+---
+
+Hello, World
+============
+
+Let's start by seeing TaskCluster in action.
+
+Visit the [Task Creator](https://tools.taskcluster/task-creator).
+Before you can submit a task, you will need to log in:
+
+ 1. Click "Log In"
+ 1. Authenticate via Persona with an email address ending in `@mozilla.com`
+ 1. Click the "Grant Permission" button, returning you to the task creator
+
+What you see in the text box is a bare-bones task description, looking something like this (the details may evolve as features are added):
+
+    { 
+      provisionerId:      'aws-provisioner-v1',
+      workerType:         'b2gtest',
+      created:            '2015-06-15Z12:00:00,
+      deadline:           '2015-06-22Z12:00:00,
+      payload: {
+        image:            'ubuntu:13.10',
+        command:          ['/bin/bash', '-c', 'echo "hello World"'],
+        maxRunTime:       60 * 10
+      },  
+      metadata: {
+        name:             "Example Task",
+        description:      "Markdown description of **what** this task does",
+        owner:            "name@example.com",
+        source:           "http://tools.taskcluster.net/task-creator/"
+      }
+    }
+
+Happily, this is already set up to print "hello world"!
+Submit the task, and click the resulting task ID to load the task inspector while the task is scheduled and run.
+
+The fields here are explained in greater detail throughout the rest of this documentation, but briefly:
+
+ * `provisionerId` identifies the TaskCluster provisioner responsible for the compute resources that will execute the task.
+   In this case, it is the [AWS Provisioner](/services/aws-provisioner/), which runs its tasks on Amazon EC2 instances using Docker.
+ * `workerType` is a parameter specific to the AWS provisioner which identifies the pool of EC2 resources within which the task should be executed.
+   Pools may use different EC2 instance types, AMIs, etc.
+
+ * `created` and `deadline` give a time boundary for the task.
+   If the task is not completed by its deadline, it will be dropped.
```

Comment:
Not true... We don't "drop" tasks we resolve them as `exception`, in this case with `reason: 'deadline-exceeded'`.